### PR TITLE
106 user ngo image upload

### PR DESCRIPTION
**Pull request by Jallow9**

### CODE REVIEW

**Comment by yuskara** [comment]
I tested and working fine

**Comment by Stef-Lev** [comment]
Also maybe we could put the Image Upload button after the Profile Button and make all the buttons a little smaller because they break in smaller screens.

**Comment by Jallow9** [comment]
Admins will not be able to upload an image to users only login users can upload image to their profiles

for the upload profile button, I will move it after the profile button.
my laptop keeps giving me problems

**Comment by Stef-Lev** [comment]
OK, looks better now. @zannis should we use some error handling so as the app doesn't break when an admin tries to upload an image, and get some feedback that he/she is not allowed to do it?

**Comment by zannis** [comment]
Thanks @Stef-Lev for reviewing this. I agree, @Jallow9  lets add a check to see if the user is an admin, and if so, hide the Upload button, to avoid such unhandled errors. Thanks!

File: frontend/challedu-app/src/components/Navigation.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
-import React, { useState } from "react";
+import React, { useState, useContext } from "react";
 import { Navbar, NavbarBrand, Nav, NavItem, Button, NavbarToggler, Collapse } from 'reactstrap';
 import '../App.css';
 import logo from '../images/logo.svg';
 import { Link } from "react-router-dom"
+import UserContrext from '../userContext';
```
Comment (Stef-Lev) [inline frontend/challedu-app/src/components/Navigation.js:6]: I didn't test it but this typo might cause problems. (userContrext)

**Pull request merged**